GenderPayGap.Core/Classes/Extensions.cs

```diff
@@ -96,15 +96,6 @@ public static DateTime GetAccountingStartDate(this SectorTypes sectorType, int y
                 size = await fileRepository.GetFileSizeAsync(filePath);
 
 
-                //Set the count in the metadata file
-                var count = 0;
-                foreach (object item in records)
-                {
-                    count++;
-                }
-
-                await fileRepository.SetMetaDataAsync(filePath, "RecordCount", count.ToString());
-

```

Comment:
I don't think we should be making product decisions on the basis that things will be awkward, if these are currently valuable features to the user. Could we capture the features on a ticket in the backlog (if they're not already) so that we can revisit them after the migration to see if there is anything we can do?

Reply:
Yeh that's fair enough. I've raised a ticket to look at this